lightningd/invoice.c

```diff
@@ -510,7 +510,7 @@ static void gossipd_incoming_channels_reply(struct subd *gossipd,
 	details = wallet_invoice_details(info, wallet, invoice);
 
 	response = json_stream_success(info->cmd);
-	json_add_hex(response, "payment_hash", details->rhash.u.u8,
+	json_add_hex(response, "payment_hash", &details->rhash,
```

Comment:
If this is common enough, maybe a botique `json_add_sha256` could be added?

Reply:
Sounds good! What about adding it in anothor PR?
It seems need change many places.